### ISSUE TITLE
Style of admonitions changed with recent updates

### ISSUE THREAD

**Issue by amotl** [crate/crate-docs-theme#555]
Hi there,

admonitions previously have been rendered like that.
![Image](https://github.com/user-attachments/assets/fd6d1bd4-12be-465b-8e3e-eeaf8f086311)

Now, they are rendered like this. [^1][^2]
![Image](https://github.com/user-attachments/assets/947403f0-04c1-4341-b7ed-18fc02bf7980)

@msbt: Is there a way to restore previous rendering, if you also like it better?

With kind regards,
Andreas.

[^1]: https://cratedb-guide--150.org.readthedocs.build/admin/clustering/scale/
[^2]: https://github.com/crate/cratedb-guide/pull/150

**Comment by amotl** [comment]
Hi again,

I forgot that we started providing two variants, discriminating between [regular](https://crate-docs-theme.readthedocs.io/en/latest/admonitions.html#regular) and [hero-style](https://crate-docs-theme.readthedocs.io/en/latest/admonitions.html#hero) admonitions.

In this case, I want to use the hero-style, and now, after actually slapping the `:class: hero` attribute on the relevant elements per https://github.com/crate/cratedb-guide/pull/150/commits/9b9c4844430d70ebdfdfb37d39b35d392241f5b4, it works as expected.

![Image](https://github.com/user-attachments/assets/f6a9423f-22b5-456a-a68d-c9fe1440c770)

Sorry for the noise.

With kind regards,
Andreas.

/cc @henrikingo

**Comment by msbt** [comment]
@amotl you're not wrong, I removed our old icons because the colors were off quite a bit and since we don't have a dedicated designer these days, that was the quickest solution. As you realized, I left the hero admonitions in place, but maybe we should adjust the line/stroke colors to match the admonitions, wdyt?

**Comment by amotl** [comment]
Hi. I think everything is all right. If you think something should be fixed or improved here, please re-open.
